app/controllers/image_uploads_controller.rb

```diff
@@ -7,7 +7,7 @@ def create
     authorize :image_upload
 
     begin
-      rate_limiter = rate_limit!
+      limit_uploads
```

Comment:
I remember @vaidehijoshi suggesting the usage of a before action. If we establish that pattern it will also be very clear for us which endpoints are rate limited (in addition to those that are rate limited inside `rack-attack`)